tests/testthat/test-read-fwf.R

```diff
@@ -127,6 +127,43 @@ test_that("error on empty spec (#511, #519)", {
   expect_error(read_fwf(txt, pos), "Zero-length.*specifications not supported")
 })
 
+# fwf_cols
+test_that("fwf_cols produces correct fwf_positions object with elements of length 2", {
+  expected <- fwf_positions(c(1,  9, 4),
```

Comment:
So this is a different style than the one in http://adv-r.had.co.nz/Style.html, which would be
```r
function_name(arg1,
              argument_name = arg2,
              ...)
```